builder/internals.go

```diff
@@ -623,7 +623,7 @@ func (b *Builder) run(c *daemon.Container) error {
 	// Wait for it to finish
 	if ret, _ := c.WaitStop(-1 * time.Second); ret != 0 {
 		return &jsonmessage.JSONError{
-			Message: fmt.Sprintf("The command %q returned a non-zero code: %d", b.Config.Cmd.ToString(), ret),
+			Message: fmt.Sprintf("The command '%s' returned a non-zero code: %d", b.Config.Cmd.ToString(), ret),
```

Comment:
I say we ignore that  :-)  Mainly because I think turning it into  \\' will look funny.
Remember, this is just an error condition/msg to help them know what failed and where - I don't think having the ' in the middle of the string will confuse them.